pkg/kf/apps/push.go

```diff
@@ -98,6 +101,28 @@ func (p *pusher) Push(appName string, opts ...PushOption) error {
 		return fmt.Errorf("failed to push app: %s", err)
 	}
 
+	merger := routes.Merger(func(newR, oldR *v1alpha1.Route) *v1alpha1.Route {
+		newR.ObjectMeta = *oldR.ObjectMeta.DeepCopy()
+
+		// Ensure the app isn't already there
+		for _, name := range oldR.Spec.KnativeServiceNames {
```

Comment:
nit: This isn't really necessary. The route webhook does a dedupe: https://github.com/google/kf/blob/master/pkg/apis/kf/v1alpha1/route_defaults.go#L30-L32

Reply:
removed this check